src/backend/shared.ts

```diff
@@ -39,6 +39,9 @@ export class CSVUtils {
       return {error: 'Header value is empty'};
     }
 
+    // Remove the headers
+    parseResult.data.shift();
```

Comment:
there was a bug where the header was duplicated in parsedCSV.headers and in the parsedCSV.records[0]